### ISSUE TITLE
A11y audit issues addressed

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#541]
With the new tooling available to check for accessibility issues, we should take a first pass at resolving those issues. As part of this issue, all issues flagged using the a11y audit tools should be addressed in one of the following ways:

- If the issue is valid, and the fix is obvious and can be addressed, then update the PF4 code to address the issue that was flagged
- If the issue is valid, and the fix is not obvious or cannot be addressed as part of this issue (e.g. it requires significant changes to the component, or further discussion before moving forward), then create an issue to capture the issue that was flagged and any blockers to addressing it as part of this PR
- If the issue is not valid, then update the audit tool to ignore that type of rule. For example, when components are tested, they are not in the context of a page layout. Therefore, warnings about proper landmark roles aren't applicable. 
  - NOTE: it might also be a valid option to modify the workspace to address some rules.

**Comment by seanforyou23** [comment]
With the demo app related issues now addressed, and new issues raised for remaining violations, I think it's safe to close this. Feel free to re-open if I've missed anything.
